### PR TITLE
Edit Profile Update password and accept site page

### LINKED ISSUE #74: Change Password modal
Integrate it with backend too

### LINKED ISSUE #75: Accept the Request made by tenant for site
Integrate the functionality of accepting or rejecting the request

### PR DESCRIPTION
**Pull request by anuj1511**

### CODE REVIEW

File: pages/auth/TenantSignUp.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ function Tenant_Signup() {
   });
 
   if (Cookies.get("userInfo")) {
-    router.push("/profile/tenant");
+    router.push("/auth/TenantSignIn");
```
Comment (rishabhborn2win) [inline pages/auth/TenantSignUp.js]: Check this if the user is logged in then why are you redirecting it to the  signin page

File: pages/editprofile/components/modal.txt [hunk 2]
```diff
@@ -0,0 +1,214 @@
+import React, { useEffect } from "react";
```
Comment (rishabhborn2win) [inline pages/editprofile/components/modal.txt]: Delete this file from the PR

File: pages/landing/components/LandingPageCard.js [hunk 3]
```diff
@@ -1,8 +1,42 @@
...
+import Cookies from "js-cookie";
+import { useSnackbar } from "notistack";
+import { useRouter } from "next/router";
+import axios from "axios";
+import React, { useContext, useEffect, useState } from "react";
 
 function LandingPageCard(props) {
+  const { enqueueSnackbar, closeSnackbar } = useSnackbar();
+
+  const acceptSite = async () => {
```
Comment (rishabhborn2win) [inline pages/landing/components/LandingPageCard.js]: There is no use of this function on this page so just remove it

File: pages/landing/components/LandingPageCard.js [hunk 4]
```diff
@@ -39,18 +73,41 @@ function LandingPageCard(props) {
...
+                            className="btn btn-success"
+                            onClick={acceptHandler}
+                          >
+                            Accept
+                          </button>
+                        </div>
+                        <div className="a-landing-card-button">
+                          <button
+                            className="btn btn-danger a-landing-card-button"
+                            onClick={rejectHandler}
```
Comment (rishabhborn2win) [inline pages/landing/components/LandingPageCard.js]: This function don't do anything why are you passing this?

File: pages/landing/components/RequestPageCards.js [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+                <span className="a-landing-card-heading"> Alias Name: </span>
+                <span className="a-landing-card-data">{props.owner}</span>
+              </h5>
+              <div className="card-text">
+                <div className="container">
+                  <LandingCardContent
+                    leftHeading="Owner"
+                    rightdata={props.owner}
+                  />
+                  <LandingCardContent leftHeading="Rent" rightdata="15000/-" />
```
Comment (rishabhborn2win) [inline pages/landing/components/RequestPageCards.js:89]: Make this rightData get updated according to the data

File: pages/profile/components/TableList.js [hunk 6]
```diff
@@ -3,25 +3,24 @@ import React from "react";
...
           <th scope="col">{props.flat}</th>
           <th scope="col">{props.loc}</th>
-          <th scope="col">{props.siteName}</th>
           <th scope="col">{props.available}</th>
           <th scope="col">{props.view}</th>
         </tr>
       </thead>
       <tbody>
-        {props.tableData?.map((data) => {
+        {detailsArray?.map((data) => {
```
Comment (rishabhborn2win) [inline pages/profile/components/TableList.js:18]: add if condition that all the selected sites are only visible in the table

**Review by rishabhborn2win — CHANGES_REQUESTED** [review]

File: pages/auth/TenantSignUp.js (hunk 1 above)
Comment (anuj1511) [reply pages/auth/TenantSignUp.js]: Sorry sir, this was a mistake

Comment (anuj1511) [reply pages/auth/TenantSignUp.js]: Updated this

File: pages/editprofile/components/modal.txt (hunk 2 above)
Comment (anuj1511) [reply pages/editprofile/components/modal.txt]: Don sir

File: pages/landing/components/LandingPageCard.js (hunk 3 above)
Comment (anuj1511) [reply pages/landing/components/LandingPageCard.js]: Yes sir deleted it

File: pages/landing/components/LandingPageCard.js (hunk 4 above)
Comment (anuj1511) [reply pages/landing/components/LandingPageCard.js]: Removed sir

File: pages/landing/components/RequestPageCards.js (hunk 5 above)
Comment (anuj1511) [reply pages/landing/components/RequestPageCards.js:89]: Done Sir

File: pages/profile/components/TableList.js (hunk 6 above)
Comment (anuj1511) [reply pages/profile/components/TableList.js:18]: Done sir

**Comment by anuj1511** [comment]
All the changes you suggested have been made

**Review by rishabhborn2win — APPROVED** [review]

**Pull request merged**